config/samples/namespaced/policy/config_v1alpha1_policy_rego.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: config.ratify.deislabs.io/v1alpha1
```

Comment:
can we add a comment to explain what is the difference between v1apha1/beta/namespaced rego, comments will help the rego sample consumers in know which sample to follow.